.github/workflows/code_quality.yml

```diff
@@ -0,0 +1,56 @@
+name: Code quality checks
```

Comment:
I'd stick to our default app template in https://github.com/maykinmedia/default-app/blob/master/.github/workflows/code_quality.yml and run it all through tox. Now the black/isort versions are not pinned, nor are the `tests` etc. code-quality checked.